network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java

```diff
@@ -55,7 +55,7 @@ public class SvgParameters {
     private EdgeInfoEnum edgeInfoDisplayed = EdgeInfoEnum.ACTIVE_POWER;
     private double pstArrowHeadSize = 8;
     private String undefinedValueSymbol = "";
-
+    private boolean highlightSubnetworks;
```

Comment:
while trying to complete the documentation I realized that this parameter is not properly named. It could be used to highlight anything else, it's only the default StyleProvider which highlights subnetworks. Could you rename it highlightGraph? or shadowGraph maybe? or do you have a better name in mind?

Reply:
highlightFeature ?